standards/collections.md

```diff
@@ -156,16 +156,17 @@ Reference the advantages of cursor-based pagination for some reasons why you may
 
 ### Guidelines
 
-- All collection-based endpoints **SHOULD** support offset-based over cursor-based pagination unless technology or performance makes cursor-based pagination more advantageous.
-- Offset-based collection endpoint responses **MUST** include the root element `paging` whenever `results` are used. 
+- While simple to implement, consider using cursor-based pagination instead
+  - Offset-based pagination does not scale as you get further down the index calculating the offset on large sets of data.
```

Comment:
keep one indentation level here as there are not multiple points... can include this in previous statement if needed as second sentence.